.github/workflows/deploy.yml

```diff
@@ -125,46 +144,105 @@ jobs:
 
       - name: Delete clock to force its recreation
         run: |
-          cf delete -f "apply-clock-${PAAS_APP_ENVIRONMENT}"
-        env:
-          CONFIRM_PRODUCTION: true
+
+          APP_NAME_SUFFIX=PAAS_APP_ENVIRONMENT
+          if [ -n "${{ github.event.inputs.pr }}" ]; then
+            APP_NAME_SUFFIX="pr-$PR_NUMBER"
+          fi;
+
+          cf delete -f "apply-clock-${APP_NAME_SUFFIX}"
```

Comment:
Shall we move this to the Makefile? It already has the logic above

Reply:
Moved in to the Makefile pending a test